com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -119,6 +145,14 @@ public PostProcessPass(RenderPassEvent evt, PostProcessData data, Material blitM
                     : GraphicsFormat.R8G8B8A8_UNorm;
                 m_UseRGBM = true;
             }
+        }
+
+        private PostProcessPass(RenderPassEvent evt, PostProcessData data)
```

Comment:
Feedback? Struct interface backcompat consequence. Better ideas?

Reply:
This is the private common part, the public api part inits the rest. This is only done to keep the old method.

Reply:
removed the backwards compatibility.